pkg/pillar/cmd/volumemgr/handleverifier.go

```diff
@@ -191,13 +167,15 @@ func updatePersistImageConfig(ctx *volumemgrContext, objType string, imageSha st
 	sub := ctx.subscription(types.VerifyImageStatus{}, objType)
 	items := sub.GetAll()
 	name := ""
+	var size int64
 	for _, s := range items {
 		status := s.(types.VerifyImageStatus)
 		if status.ImageSha256 == imageSha {
 			log.Infof("Adding RefCount %d from %s to %s",
 				status.RefCount, status.ImageID, imageSha)
 			refcount += status.RefCount
 			name = status.Name
+			size = status.Size
```

Comment:
Since you do that, why don't you get the filelocation from status as well and avoid the extra argument?

Also, both of those should have a comment saying that we assume that the name, size, and filelocation are the same because it is the same sha/object.

Reply:
Changed.